Formula/s/stashapp.rb

```diff
@@ -0,0 +1,28 @@
+class Stashapp < Formula
+  desc "Organizer for your porn"
```

Comment:
```suggestion
  desc "Organizer for your explicit images"
```
Perhaps just to avoid any of our pages getting hit by filters?